### PR TITLE
i18n(zh-cn): translate guides/migrate-to-astro/from-pelican.mdx

### PR DESCRIPTION
**Pull request by HawtinZeng**
#### What kind of changes does this PR include?

- Translated content

#### Description

- Translate guides/migrate-to-astro/from-pelican.mdx into Chinese.

### CODE REVIEW

**Comment by HawtinZeng** [comment]
@liruifengv, the translation is finished!

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+description: 从一个现有的 Pelican 项目迁移到 Astro 的方法
+type: migration
+stub: true
+framework: Pelican
+i18nReady: true
+---
+
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+[Pelican](https://getpelican.com) 是一个开源的使用Python构建的静态网站生成器
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx:12]:
```suggestion
[Pelican](https://getpelican.com) 是一个开源的使用 Python 构建的静态网站生成器。
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+## Pelican与Astro的主要相似点
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx:14]:
```suggestion
## Pelican 与 Astro 的主要相似点
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+Pelican和Astro的一些相似点会帮助你完成迁移。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx:16]:
```suggestion
Pelican 和 Astro 的一些相似点会帮助你完成迁移。
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+- Pelican和Astro都是静态网站生成器，尤其适合[专注于内容的网站](/zh-cn/concepts/why-astro/#content-focused)，例如博客。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx:18]:
```suggestion
- Pelican 和 Astro 都是静态网站生成器，尤其适合[专注于内容的网站](/zh-cn/concepts/why-astro/#以内容为中心)，例如博客。
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+- Pelican和Astro都内置支持[Markdown文档](/zh-cn/guides/content/#markdown-authoring)，包括YAML的前置信息(frontmatter properties)。但是Astro中预留的前置信息比Pelican少得多。尽管很多Pelican内生效的前置属性信息在Astro中并不会生效，但你仍然可以使用使用Markdown和前置信息。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx:20]:
```suggestion
- Pelican 和 Astro 都内置支持 [Markdown文档](/zh-cn/guides/content/#markdown-创作)，包括页面元数据的 frontmatter YAML 属性。但是 Astro 中预留的 frontmatter 比 Pelican 少得多。尽管很多 Pelican 内生效的 frontmatter 属性在 Astro 中并不会生效，但你仍然可以使用使用 Markdown 和 frontmatter 。
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+## Pelican与Astro的主要差异点
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx:22]:
```suggestion
## Pelican 与 Astro 的主要差异点
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+当使用Astro重新构建Pelican网站时，你会注意到一些重要的差异。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx:24]:
```suggestion
当使用 Astro 重新构建 Pelican 网站时，你会注意到一些重要的差异。
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+- Pelican支持Markdown和reStructured Text(`.rst`)。Astro可以[从Markdown和MDX文件创建页面](/zh-cn/guides/markdown-content/)，但是不支持reStructured Text。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx:26]:
```suggestion
- Pelican 支持 Markdown 和 reStructured Text(`.rst`)。Astro 可以[从Markdown和MDX文件创建页面](/zh-cn/guides/markdown-content/)，但是不支持 reStructured Text。
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+
+- Pelican使用HTML格式和Jinja语法创建模板，而[Astro语法](/zh-cn/core-concepts/astro-components/)是一个类似于JSX的HTML超集，也就是说任何的HTML都符合Astro语法规则。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx:28]:
```suggestion
- Pelican 使用 HTML 格式和 Jinja 语法创建模板，而 [Astro 语法](/zh-cn/core-concepts/astro-components/)是一个类似于 JSX 的 HTML 超集，也就是说任何的 HTML 都符合 Astro 语法规则。
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+
+- Pelican为构建内容丰富的网站而设计，例如博客，它的博客特性Astro是不提供的，你必须自己构建。但是，Astro也会提供一些特性包括[官方博客主题](https://github.com/withastro/astro/tree/latest/examples/blog)。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx:30]:
```suggestion
- Pelican 为构建内容丰富的网站而设计，例如博客，它的博客特性 Astro 是不提供的，你必须自己构建。但是，Astro 也会提供一些特性包括[官方博客主题](https://github.com/withastro/astro/tree/latest/examples/blog)。
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+
+## 从Pelican迁移到Astro的步骤
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx:32]:
```suggestion
## 从 Pelican 迁移到 Astro 的步骤
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+
+为了将Pelican文档转换成Astro，你可以从我们的[星空文档主题模板](https://starlight.astro.build)开始，或者从[主题案例](https://astro.build/themes/)里面的社区主题开始。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx:34]:
```suggestion
为了将 Pelican 文档转换成 Astro，你可以从我们的 [StarLight 文档主题模板](https://starlight.astro.build)开始，或者从[主题案例](https://astro.build/themes/)里面的社区主题开始。
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx [hunk 13]
```diff
@@ -0,0 +1,64 @@
...
+
+你可以通过传入`--template`参数到`create astro`命令开启一个Astro项目，并选择我们提供的某个模板。或者，你也可以从一个[Astro Github仓库](/en/install/auto/#starter-templates)开始你的项目。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx:36]:
```suggestion
你可以通过传入`--template`参数到`create astro`命令开启一个 Astro 项目，并选择我们提供的某个模板。或者，你也可以从一个 [Astro Github 仓库](/zh-cn/install/auto/#入门模板)开始你的项目。
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx [hunk 14]
```diff
@@ -0,0 +1,64 @@
...
+    ```
+    </Fragment>
+    <Fragment slot="yarn">
+    ```shell
+    yarn create astro --template starlight
+    ```
+    </Fragment>
+  </PackageManagerTabs>
+
+导入你现有的Markdown文档[创建Markdown页面](/en/guides/markdown-content/)。你可以将Pelican`content/`文件夹中的文档复制到Astro中的`src/pages/`文件夹，从而开启基于文件的路由。你或许会想阅读[Astro项目结构](/zh-cn/core-concepts/project-structure/)去了解什么文件应该放置于什么位置。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx:56]:
```suggestion
导入你现有的 Markdown 文档[创建 Markdown 页面](/zh-cn/guides/markdown-content/)。你可以将Pelican`content/`文件夹中的文档复制到 Astro 中的 `src/pages/` 文件夹，从而开启基于文件的路由。你或许会想阅读 [Astro 项目结构](/zh-cn/core-concepts/project-structure/)去了解什么文件应该放置于什么位置。
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx [hunk 15]
```diff
@@ -0,0 +1,64 @@
...
+
+Pelican会帮你处理网站的布局和元数据。你或许会想阅读[创建Astro布局包裹Markdown](/en/core-concepts/layouts/#markdownmdx-layouts)从而了解如何在Astro中自己管理模板，包括页面的`<head>`。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx:58]:
```suggestion
Pelican 会帮你处理网站的布局和元数据。你或许会想阅读[创建 Astro 布局封装 Markdown](zh-cn/core-concepts/layouts/#markdownmdx-布局) 从而了解如何在 Astro 中自己管理模板，包括页面的`<head>`。
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx [hunk 16]
```diff
@@ -0,0 +1,64 @@
...
+
+为了迁移其它类型的网站，例如一个作品集或者一个博客，请进入[astro.new](https://astro.new)查看更多的网站模板，你会找到模板对应的Github仓库链接和在线开发环境链接(StackBlitz, CodeSandbox and Gitpod)。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx:60]:
```suggestion
为了迁移其它类型的网站，例如一个作品集或者一个博客，请进入 [astro.new](https://astro.new) 查看更多的网站模板，你会找到模板对应的 GitHub 仓库链接和在线开发环境链接(StackBlitz, CodeSandbox and Gitpod)。
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx [hunk 17]
```diff
@@ -0,0 +1,64 @@
...
+
+## 社区资源
+
+- 自己添加
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx:64]:
```suggestion
- 添加你自己的！
```

**Review by liruifengv — CHANGES_REQUESTED** [review]
Thanks for your contribution! @HawtinZeng There has some need to change. Please update and then call me to review.

**Comment by liruifengv** [comment]
hello。这个还有一些建议没有修改

**Comment by HawtinZeng** [comment]
好了，你再看看，应该没问题了 @liruifengv

**Review by liruifengv — APPROVED** [review]
LGTM

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx [hunk 18]
```diff
@@ -0,0 +1,64 @@
...
+    <Fragment slot="yarn">
+    ```shell
+    yarn create astro --template starlight
+    ```
+    </Fragment>
+  </PackageManagerTabs>
+
+导入你现有的 Markdown 文档[创建 Markdown 页面](/zh-cn/guides/markdown-content/)。你可以将Pelican `content/`文件夹中的文档复制到 Astro 中的 `src/pages/` 文件夹，从而开启基于文件的路由。你或许会想阅读 [Astro 项目结构](/zh-cn/core-concepts/project-structure/)去了解什么文件应该放置于什么位置。
+
+Pelican 会帮你处理网站的布局和元数据。你或许会想阅读[创建 Astro 布局封装 Markdown](zh-cn/core-concepts/layouts/#markdownmdx-布局) 从而了解如何在 Astro 中自己管理模板，包括页面的`<head>`。
```
Comment (Yan-Thomas) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx:58]:
````suggestion
Pelican 会帮你处理网站的布局和元数据。你或许会想阅读[创建 Astro 布局封装 Markdown](/zh-cn/core-concepts/layouts/#markdownmdx-布局) 从而了解如何在 Astro 中自己管理模板，包括页面的`<head>`。
```

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks for the great work, and welcome to Team Docs!

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx (hunk 18 above)
Comment (HawtinZeng) [reply src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx]: Why add three backticks? I am a little confused.

Comment (Yan-Thomas) [reply src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx]: > Why add three backticks? I am a little confused.

Oh no, that was an issue in GitHub's suggestion syntax I am afraid, would you like to open a PR removing this?

Comment (HawtinZeng) [reply src/content/docs/zh-cn/guides/migrate-to-astro/from-pelican.mdx]: Ok

**Pull request merged**